tools/DB_date_zeros_removal.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Script exporting Update statements to remove 0000-00-00 values
+ * and replace them by NULL
+ *
+ * PHP Version 5
+ *
+ * @category Main
+ * @package  Loris
+ * @author   Various <example@example.com>
```

Comment:
isn't Rida the author? We should maybe standardize on something for the author line across all of standard LORIS, but right now I think everything just has the original author.